cmd/cape/cmd/run.go

```diff
@@ -127,9 +127,15 @@ func run(cmd *cobra.Command, args []string) error {
 		input = buf.Bytes()
 	}
 
-	authType := entities.AuthenticationTypeAuth0
-	if functionToken != "" {
+	authType := entities.AuthenticationTypeFunctionToken
```

Comment:
Can we make auth0 the default again somehow? I still prefer that function auth is the default in code

Reply:
> I still prefer that function auth is the default in code

Do you mean auth0? This is defaulting to function auth